### PR TITLE
Rename LiveBlogChromeNotificationsProd

### PR DESCRIPTION
**Pull request by janua**
We ran `LiveBlogChromeNotificationsProd` for a while under a 1% test. Bumping the test to 10% only got new users that never seen the test before into the 10%, which wasn't as much as we would like.

This renames the test therefore rebooting it to 10% of all users.

@NathanielBennett @crifmulholland @gtrufitt

### CODE REVIEW

**Comment by JonNorman** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/experiments/tests/live-blog-chrome-notifications-prod.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ define([
 ) {
     return function () {
 
-        this.id = 'LiveBlogChromeNotificationsProd';
+        this.id = 'LiveBlogChromeNotificationsProd2';
         this.start = '2016-06-08';
```
Comment (JonNorman) [inline static/src/javascripts/projects/common/modules/experiments/tests/live-blog-chrome-notifications-prod.js]: I suppose it doesn't matter but might you want to alter the start time?

**Comment by gtrufitt** [comment]
I think you should also update the switch name

File: static/src/javascripts/projects/common/modules/experiments/tests/live-blog-chrome-notifications-prod.js (hunk 1 above)
Comment (NathanielBennett) [inline static/src/javascripts/projects/common/modules/experiments/tests/live-blog-chrome-notifications-prod.js]: I think you'll also need to change this line:
https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/liveblog.js#L71

**Comment by NathanielBennett** [comment]
:+1: Apart from comments

**Comment by janua** [comment]
@NathanielBennett `liveblog.js` has been updated.
I have also removed the `internal` test, we no longer use it or need it.

@gtrufitt We don't need to update the switch name.

**Comment by gtrufitt** [comment]
Sure. Just from a cleanliness point of view than a requirement. 👍

**Pull request merged**